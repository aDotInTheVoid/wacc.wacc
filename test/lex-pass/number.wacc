2
-12
1234134