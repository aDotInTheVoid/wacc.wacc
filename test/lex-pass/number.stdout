1 number 2
1 minus -
2 number 12
7 number 1234134
0 eof 
